airbyte-integrations/connectors/source-mssql/src/testFixtures/java/io/airbyte/integrations/source/mssql/MsSQLTestDatabase.java

```diff
@@ -181,7 +222,10 @@ public MsSQLConfigBuilder configBuilder() {
   static public class MsSQLConfigBuilder extends ConfigBuilder<MsSQLTestDatabase, MsSQLConfigBuilder> {
 
     protected MsSQLConfigBuilder(MsSQLTestDatabase testDatabase) {
+
       super(testDatabase);
+      with(JdbcUtils.JDBC_URL_PARAMS_KEY, "loginTimeout=2");
```

Comment:
this allows us to reduce the test running times from 1m to 2seconds for the tests that are expecting a failure to connect to the DB!

Reply:
Do we always want to set this for all test cases, though? I don't have a problem with it. I took a more conservative approach with `maybeSetShorterConnectionTimeout` but that conservatism may not have been warranted.

Reply:
no problems with it so far...